typescript/commands/signature.py

```diff
@@ -60,7 +60,7 @@ def on_selected(self, index):
 
 class TypescriptSignaturePopup(sublime_plugin.TextCommand):
     def is_enabled(self):
-        return TOOLTIP_SUPPORT and is_typescript(self.view)
+        return TOOLTIP_SUPPORT and is_typescript(self.view) and get_language_service_enabled()
```

Comment:
That doesn't seem to have much gain over what it is right now though. It might be worthwhile to refactor all the labels later.